x-pack/plugin/core/src/main/resources/monitoring-es.json

```diff
@@ -1,11 +1,11 @@
 {
   "index_patterns": [ ".monitoring-es-${monitoring.template.version}-*" ],
-  "version": 7140099,
+  "version": 8000099,
   "settings": {
     "index.number_of_shards": 1,
     "index.number_of_replicas": 0,
     "index.auto_expand_replicas": "0-1",
-    "index.format": 7,
+    "index.format": 8,
```

Comment:
Will this conflict with https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/core/src/main/resources/monitoring-es-mb.json#L2 ? I think the beats template is already `8`. @jbaiera might be able to help confirm.